docs/guides/admin/docs/workflowoperationhandlers/speechtotext-woh.md

```diff
@@ -10,13 +10,16 @@ The speech to text operation can be used to generate subtitles for Videos or Aud
 engines available, [Whisper](../modules/transcription.modules/whisper.md) and 
 [Vosk](../modules/transcription.modules/vosk.md). The subtitles file format ist WebVTT.
 
+This operation is designed to only work with one source file per flavor. If you want to use multiple source files in one
+flavor, you need to set the same tag to each element.
 
 Parameter Table
 ---------------
 
 | configuration keys | required | Example           | description                                                                                                                                        |
 |--------------------|----------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
 | source-flavor      | yes      | source/presenter  | The source media package to use                                                                                                                    |
+| source-tags        | no       | 1080p             | The tags that the source track uses                                                                                                                   |
```

Comment:
> The tags that the source track uses

Sounds like I need to specify ALL tags of a particular track, but actually I just need to specify one of them, correct?

Reply:
thats correct, you need only to check one tag, not all.

Reply:
For example. If you have 2 tracks with tags `"one","two"` and another with the tags `"one","three"`

If you target only `one`, it will process both.
If you target `one,two`, it will process the first.